adapter.go

```diff
@@ -164,17 +172,9 @@ func NewAdapterByDBUseTableName(db *gorm.DB, prefix string,tablename string) (*A
 	return a, nil
 }
 
+// NewAdapterByDB obtained through an existing Gorm instance get  a adapter
```

Comment:
`obtained through an existing Gorm instance get  a adapter`

This sentense doesn't make sense. Change it to smooth English.